### ISSUE TITLE
FMW upgraded from 5.0.6 to 5.0.8 hangs on start on Win10

### ISSUE THREAD

**Issue by kparal** [FedoraQt/MediaWriter#662]
If I install FMW 5.0.6 (either from here from github, or from getfedora.org), and then download and install FMW 5.0.8 over it (without uninstalling 5.0.6 first), FMW doesn't start, it hangs during startup (I don't know where to get logs). Rebooting doesn't help, and uninstalling and reinstalling FMW doesn't help either (!!!).

I believe this is somehow connected to a clash between installation files of 5.0.6 and 5.0.8. Because when I uninstall 5.0.8 (installed over 5.0.6), it leaves a lot of files on disk in `C:\Program Files (x86)\Fedora Media Writer`. That's probably why reinstalling doesn't help, because the "invalid" files are never removed.

Reproducer:
1. Install FMW 5.0.6
2. Start FMW, it works. Close it.
3. Install FMW 5.0.8.
4. Start FMW, it doesn't launch at all.
5. Uninstall FMW, install 5.0.8 again.
6. Goto 4.

Workaround:
1. Kill mediawriter.exe in the task manager.
2. Uninstall FMW.
3. Delete `C:\Program Files (x86)\Fedora Media Writer`
4. Install FMW 5.0.8, now it finally works.

Overall this looks like to be a big problem, because most people will not uninstall 5.0.6 first (that's not the usual workflow), and therefore will be affected by this bug, and they'll be unable to launch FMW at all.

Let's make sure Fedora doesn't publish 5.0.8 on their website until this problem is resolved.

**Comment by RileyMarquis3** [comment]
@kparal is correct, based on my attempt to update to FMW 5.0.8.  Is the issue the failure to delete the files, or is it a combination of x86 and 64 bit files, with the older versions of FMW being 32 bit and not being removed?

**Comment by grulja** [comment]
> @kparal is correct, based on my attempt to update to FMW 5.0.8. Is the issue the failure to delete the files, or is it a combination of x86 and 64 bit files, with the older versions of FMW being 32 bit and not being removed?

Good point. I didn't realize we now make 64bit version instead of 32bit as we do now starting with FMW 5.0.7. I will have to look to figure out how to remove the original content before installing the new one.
